app/src/test/java/org/oppia/app/testing/player/split/PlayerSplitScreenTesting.kt

```diff
@@ -0,0 +1,83 @@
+package org.oppia.app.testing.player.split
+
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.FirebaseApp
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.player.SplitScreenManager
+import org.oppia.app.testing.ExplorationTestActivity
+import org.robolectric.annotation.Config
+
+// Devices reference: https://material.io/resources/devices/
+@RunWith(AndroidJUnit4::class)
+class PlayerSplitScreenTesting {
+
+  @Before
+  @ExperimentalCoroutinesApi
+  fun setUp() {
```

Comment:
Please add an issue and mention that issue number in this PR. The issue will track to add test cases for `ImageClickInput`

Reply:
Please mention that issue number with proper comment in this file. like
`// TODO(#XXX): Add test cases for ImageClickInput  interaction for sliptview screen.`

Reply:
Done